com.vrcfury.vrcfury/Editor/VF/Feature/ToggleBuilder.cs

```diff
@@ -26,20 +26,36 @@ public class ToggleBuilder : FeatureBuilder<Toggle> {
     private VFABool param;
     private AnimationClip restingClip;
 
+    private bool addMenuItem;

```

Comment:
Typically bad to pass values between methods using class properties. I abuse it to pass between separate actions, but within an action they should just be passed as parameters.